it/api/endpoints/lossClaim/list/v3/ListLossClaimsControllerISpec.scala

```diff
@@ -148,17 +143,41 @@ class ListLossClaimsControllerISpec extends V3IntegrationBaseSpec {
     }
   }
 
+  private trait NonTysTest extends Test {
+    override def taxYear: Option[String] = Some("2019-20")
+
+    override def downstreamUrl: String = s"/income-tax/claims-for-relief/$nino"
+
+    override def queryParams: Seq[(String, String)] =
```

Comment:
We can take out `queryParams` here and put it in `Test` instead of `NonTysTest`, as we are overriding taxYear in each trait, so the taxYear QP will be different for each trait

Reply:
Then maybe rename 'queryParams' to 'mtdQueryParams` as it is bit confusing at first glance whether they are for downstream or mtd